client/.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true 
```

Comment:
Just curious, why is this needed? Or was it a default setting/file?

Reply:
This was needed in order for npm start to work due to that dependency issue I was running into. I tried deleting the node modules and reinstalling and also following all the listed steps and neither of those were working. This was the only solution I could find.